deploy.sh

```diff
@@ -39,13 +36,7 @@ echo "aws_access_key_id = $AWS_ACCESS_ID" >> ~/.aws/config
 echo "aws_secret_access_key = $AWS_SECRET_KEY" >> ~/.aws/config
 
 # Add environment variables
-eb setenv FLASK_ENVIRONMENT_CONFIG=$FLASK_ENVIRONMENT_CONFIG 
-eb setenv MAIL_DEFAULT_SENDER=$MAIL_DEFAULT_SENDER 
-eb setenv MAIL_SERVER=$MAIL_SERVER 
-eb setenv APP_MAIL_USERNAME=$APP_MAIL_USERNAME 
-eb setenv APP_MAIL_PASSWORD=$APP_MAIL_PASSWORD 
-eb setenv SECRET_KEY=$SECRET_KEY 
-eb setenv SECURITY_PASSWORD_SALT=$SECURITY_PASSWORD_SALT
+eb setenv FLASK_ENVIRONMENT_CONFIG=$FLASK_ENVIRONMENT_CONFIG MAIL_DEFAULT_SENDER=$MAIL_DEFAULT_SENDER MAIL_SERVER=$MAIL_SERVER APP_MAIL_USERNAME=$APP_MAIL_USERNAME APP_MAIL_PASSWORD=$APP_MAIL_PASSWORD SECRET_KEY=$SECRET_KEY SECURITY_PASSWORD_SALT=$SECURITY_PASSWORD_SALT
```

Comment:
is this how it should be? inline instead of each env inline?

Reply:
This is a better way of doing this. 
`eb setenv` command sets the environment variable which makes changes in the AWS instance (takes 2 mins approx).

According to the earlier approach, it would take 2 mins for each `eb setenv`. Now It will only take time for one and all values will be set.

Reply:
Wow @m-murad  thanks for explaining it! That change makes total sense!